src/components/Character.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Character = ({ _id, name, photoUrl, toDetailPage }) => {
+  return (
+    <button value={_id} onClick={toDetailPage}>
```

Comment:
probably wouldnt make a function and pass it all the way down here. instead of having a button and calling that function. you could wrap the img and h1 inside a Link tag from react-router-dom and use the "to" attribute to send you to a new url.